### PR TITLE
Apply prettier to markdown files

### LINKED ISSUE #10: Format all markdown files with prettier?
https://prettier.io/

It supports markdown and has many IDE/editors integration.

### PR DESCRIPTION
**Pull request by taoky**
- Used `prettier` to rewrite all markdown files.
- Added `prettier` to GitHub Actions CI.

Fixes #10

### CODE REVIEW

File: .github/workflows/build-test.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+name: build-test
+
+on:
+  push:
+    branches-ignore:
+      - master
```
Comment (zhsj) [inline .github/workflows/build-test.yml:6]: This should be on pull-request. Your PR doesn't trigger any test.

**Review by zhsj — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -25,11 +25,29 @@ jobs:
...
+          ${{ runner.os }}-
     - name: Install dependencies
       run: |
         sudo apt-get -y install libgsl-dev
         bundle install --path=vendor/bundle
         bundle clean
+        npm install
+    - name: Prettier check
+      run: |
+        npx prettier --check .
```
Comment (zhsj) [inline .github/workflows/build.yml:50]: when deploying on master branch, the syntax lint is redundant.

**Comment by taoky** [comment]
When writing workflows what I thought is that `build-test.yml` is for testing new features/changes (and should not trigger site deployment), and `build.yml` is for small changes (like fixing typos, adding new activity in the table, etc.) and deploying, so I set prettier check on both workflows, and ignores `master` branch in `build-test.yml`.

One method is to:

- `build-test.yml`: testing on all pushes (including `master`) and pull requests
- `build.yml`: deploying on master branch (even when `build-test.yml` fails)

But after reviewing my own code again just now, I found that `build-test.yml` and `build.yml` have a lot of duplicated code lines, probably I should combine them into one:

- Testing on all pushes and pull requests
- Only running deployment when it's on master branch

It should be better.

File: .github/workflows/build.yml (hunk 2 above)
Comment (taoky) [reply .github/workflows/build.yml:53]: Now linting and deploying are all in `build.yml`. `build-test.yml` is removed in the latest commit.

**Review by zhsj — APPROVED** [review]

**Review by iBug — APPROVED** [review]
LGTM

**Pull request merged**